dapps/react-dapp-v2/src/chains/polkadot.ts

```diff
@@ -29,15 +51,15 @@ export function getChainMetadata(chainId: string): ChainMetadata {
 export function getChainRequestRender(
   request: JsonRpcRequest
 ): ChainRequestRender[] {
-  let params = [{ label: "Method", value: request.method }];
+  let params = [{ label: 'Method', value: request.method }];
 
   switch (request.method) {
     default:
       params = [
         ...params,
         {
-          label: "params",
-          value: JSON.stringify(request.params, null, "\t"),
+          label: 'params',
+          value: JSON.stringify(request.params, null, '\t'),
```

Comment:
Looks like something happened with string formatting in this file specifically? The other diffs still use double quote `""` so not sure what happened 😄

Reply:
It was my prettier which was set to single quote:) fixed.